external-crates/move/crates/move-compiler/src/typing/macro_expand.rs

```diff
@@ -37,6 +37,7 @@ struct Context<'a, 'b> {
     by_name_args: ArgMap,
     tparam_subst: TParamSubst,
     macro_color: Color,
+    lambda_labels: u16,
```

Comment:
Need some way to generate unique labels for blocks representing lambdas - open to suggestions other than what I used here!

Reply:
Why do they need to be unique?

Reply:
> Why do they need to be unique?

Otherwise, if we use two lambdas in the same macro, the later stages of the compiler barf. "

> The lambda block's color is already unique

Which one is this? Neither `context.macro_color` nor `use_fun_color` in the code below initializing the label are unique.